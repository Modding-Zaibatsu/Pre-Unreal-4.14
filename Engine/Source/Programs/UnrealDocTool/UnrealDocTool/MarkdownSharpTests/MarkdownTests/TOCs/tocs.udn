[TOC]

Beginning paragraph.

# Header 1 Level 1

Paragraph between.

Setext header 2 Level 2
-----------------------

Paragraph between with TOC.

[TOC]

## Header 3 Level 2

Paragraph between.

### Header 4 Level 3

Paragraph between.

Setext header 5 Level 1
=======================

Ending paragraph.