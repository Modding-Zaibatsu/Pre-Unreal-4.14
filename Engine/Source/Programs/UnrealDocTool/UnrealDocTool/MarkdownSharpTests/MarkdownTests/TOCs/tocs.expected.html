<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" dir="ltr" lang="en-US">
    <head>
    </head>
    <body>
        <ul>
            <li><a href="#header1level1">Header 1 Level 1</a></li>
            <ul>
                <li>
                    <a href="#setextheader2level2">Setext header 2 Level 2</a>
                </li>
                <li>
                    <a href="#header3level2">Header 3 Level 2</a>
                </li>
                <ul>
                    <li>
                        <a href="#header4level3">Header 4 Level 3</a>
                    </li>
                </ul>
            </ul>
            <li>
                <a href="#setextheader5level1">Setext header 5 Level 1</a>
            </li>
        </ul>

        <p>Beginning paragraph.</p>

        <h1 id="header1level1">Header 1 Level 1</h1>

        <p>Paragraph between.</p>

        <h2 id="setextheader2level2">Setext header 2 Level 2</h2>

        <p>Paragraph between with TOC.</p>

        <ul>
            <li>
                <a href="#header1level1">Header 1 Level 1</a>
            </li>
            <ul>
                <li>
                    <a href="#setextheader2level2">Setext header 2 Level 2</a>
                </li>
                <li>
                    <a href="#header3level2">Header 3 Level 2</a>
                </li>
                <ul>
                    <li>
                        <a href="#header4level3">Header 4 Level 3</a>
                    </li>
                </ul>
            </ul>
            <li>
                <a href="#setextheader5level1">Setext header 5 Level 1</a>
            </li>
        </ul>

        <h2 id="header3level2">Header 3 Level 2</h2>

        <p>Paragraph between.</p>

        <h3 id="header4level3">Header 4 Level 3</h3>

        <p>Paragraph between.</p>

        <h1 id="setextheader5level1">Setext header 5 Level 1</h1>

        <p>Ending paragraph.</p>
    </body>
 </html>
